src/asgen/utils.d

```diff
@@ -370,6 +374,10 @@ body
             downloader.dataTimeout = dur!"seconds" (30);
             downloader.onReceive = (data) => onReceiveCb (dest, data);
             downloader.perform();
+            if ("last-modified" in downloader.responseHeaders) {
```

Comment:
I *hope* that hashing is super fast (and maybe this could be optimised anyway), so this shouldn't really matter...